webapp/components/Comment.vue

```diff
@@ -11,7 +11,6 @@
   </div>
   <div v-else :class="{ comment: true, 'disabled-content': comment.deleted || comment.disabled }">
     <ds-card>
-      <ds-space margin-bottom="small" />
       <ds-space margin-bottom="small">
         <hc-user :user="author" :date-time="comment.createdAt" />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-24T19:43:55Z" title="Monday, June 24th 2019, 9:43:55 pm +02:00">Jun 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -23,10 +23,27 @@
     <!-- eslint-disable vue/no-v-html -->
     <!-- TODO: replace editor content with tiptap render view -->
     <ds-space margin-bottom="small" />
-    <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
-    <!-- eslint-enable vue/no-v-html -->
+    <div v-if="!isActive" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
```


The logic implemented in this template is calling for it's own component. How about `<collapsible>`?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-03T06:35:09Z" title="Wednesday, July 3rd 2019, 8:35:09 am +02:00">Jul 3, 2019</time>_

---


We have to refactor this anyway by using the editor library to display the content very soon. We should do this than together …